server/service/src/document/update_trigger.rs

```diff
@@ -0,0 +1,106 @@
+use std::str::FromStr;
+
+use chrono::NaiveDate;
+use repository::{
+    Document, Gender as NameRowGender, NameRow, NameRowRepository, NameStoreJoinRepository,
+    NameStoreJoinRow, NameType, StorageConnection,
+};
+
+extern crate schemafy_core;
+extern crate serde;
+extern crate serde_json;
+
+use serde::{Deserialize, Serialize};
+use util::uuid::uuid;
+
+use super::document_service::DocumentInsertError;
+
+schemafy::schemafy!("src/document/schemas/patient.json");
```

Comment:
yeah repo is not super active but it has a couple of users and contributors. Since we only need it for the well known types it should IMHO be ok to use, i.e. we would encounter issues with it early on.